<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Model to DXF</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../py_widget/resource.qrc">
    <normaloff>:/safe/Resources/icons/dxf.svg</normaloff>:/safe/Resources/icons/dxf.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="columns_checkbox">
     <property name="text">
      <string>Columns</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="punches_checkbox">
     <property name="text">
      <string>Punches</string>
     </property>
     <property name="icon">
      <iconset resource="../../py_widget/resource.qrc">
       <normaloff>:/safe/Resources/icons/punch.svg</normaloff>:/safe/Resources/icons/punch.svg</iconset>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="filename">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>*.dxf</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="export_button">
       <property name="text">
        <string>Export</string>
       </property>
       <property name="icon">
        <iconset resource="../../py_widget/resource.qrc">
         <normaloff>:/safe/Resources/icons/dxf.svg</normaloff>:/safe/Resources/icons/dxf.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_pushbutton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../py_widget/resource.qrc">
         <normaloff>:/safe/Resources/icons/cancel.png</normaloff>:/safe/Resources/icons/cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../py_widget/resource.qrc"/>
 </resources>
 <connections/>
</ui>
